code/atom.dm

```diff
@@ -615,6 +615,8 @@
 //mbc : sorry, i added a 'is_special' arg to this proc to avoid race conditions.
 /atom/proc/attackby(obj/item/W as obj, mob/user as mob, params, is_special = 0)
 	if (user && W && !(W.flags & SUPPRESSATTACK))  //!( istype(W, /obj/item/grab)  || istype(W, /obj/item/spraybottle) || istype(W, /obj/item/card/emag)))
+		if (SEND_SIGNAL(src,COMSIG_ATTACKBY,W,user) & COMSIGBIT_ATTACKBY_COMPLETE ? 0 : 1)
```

Comment:
The ` ? 0 : 1` is needed, otherwise it's falsey.
context: `#define COMSIGBIT_ATTACKBY_COMPLETE 1`

Reply:
Wait, I really fucked this up